bc_obps/registration/migrations/0112_prod_fix_contacts_post_transfer.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 5.0.14 on 2025-05-12 20:10
+
+from django.db import migrations
+
+"""
+One-time forward-only migration to be applied to prod data.
+Purpose: delete operation representatives from operation_contacts table for operations that have been transferred to a new operator.
```

Comment:
I think the PR description is clearer, we could resuse: `Forward-only prod data migration to remove the contacts associated with an operation that has been transferred, if the contact is associated with the from_operator (not the to_operator) of the transfer.`